lib/src/server.dart

```diff
@@ -61,17 +61,28 @@ class Server {
   /// invoked. If it is not set, the exception will be swallowed.
   final ErrorCallback onUnhandledError;
 
+  /// If true, this [Server] will accept some requests which are not conformant
+  /// with the JSON-RPC 2.0 specification. In particular, requests missing the
+  /// `jsonrpc` parameter will be accepted.
```

Comment:
This looks backwards to me. We should also format with a header.
```suggestion
  /// Whether to strictly enforce the JSON-RPC 2.0 specification for received messages.
  ///
  /// If `false`, this [Server] will accept some requests which are not conformant
  /// with the JSON-RPC 2.0 specification. In particular, requests missing the
  /// `jsonrpc` parameter will be accepted.
```

Reply:
Done. Not sure what you mean about "format with a header"?

Reply:
I mean having a summary sentence, then a blank line.

https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph